core/frontend/src/tile/map/MapLayerImageryProvider.ts

```diff
@@ -268,35 +295,48 @@ export abstract class MapLayerImageryProvider {
     return true;
   }
 
-  /** @internal */
-  // calculates the projected x cartesian coordinate in EPSG:3857from the longitude in EPSG:4326 (WGS84)
+  /**
```

Comment:
Do we really want to expose those reprojection methods?

Reply:
They are used by ArcGISFeatureProvider in the map-layers-formats package, so I thought they should be promoted due to being used outside of core-frontend and maybe being useful in the future for new imagery providers/formats. But promoting them now might be premature so I can revert until they are needed by more formats